app/views/admin/suppliers/supplier_users/index.html.erb

```diff
@@ -1,3 +1,7 @@
+<% if @user_deleted %>
+  <%= govuk_panel title: "User Deleted", body: nil %>
```

Comment:
This should be a notification banner as well

Reply:
Updated to notification flash - Checked with Shani